2019/pt-br/src/0xa9-improper-assets-management.md

```diff
@@ -0,0 +1,54 @@
+API9:2019 Gerenciamento de Ativos Inapropriado
+==============================================
+
+| Agentes/Vetores | Fraquezas de Segurança | Impactos |
+| - | - | - |
+| Específico da API : Explorabilidade **3** | Prevalência **3** : Detecção **2** | Técnico **2** : Específico do negócio |
+| Versões desatualizadas de APIs geralmente carecem de *patches* e são um meio fácil de comprometer sistemas sem a presença de mecanismos de segurança no estado-da-arte, e que podem existir também para proteger versões atualizadas de APIs. | Documentação desatualizada torna mais difícil encontrar e/ou corrigir vulnerabilidades. Falta de um inventário de ativos e estratégias de retirada levam ao cenário de sistemas sem atualização permanecerem em execução, podendo resultado no vazamento de dados sensíveis. É bastante comum encontrar APIs expostas sem necessidade em razão de conceitos modernos como os de microsserviços, que permite que aplicativos sejam lançados independentemente (ex.: computação em nuvem, kubernetes). | Atacantes ter acesso a dados sensíveis e até tomar o controle de servidores por meio de uma velha e desatualizada API conectada ao mesmo banco de dados. |
```

Comment:
fix
* where it's written "_(...) podendo **resultado** no vazamento de dados sensíveis._"
* it should be "_(...) podendo **resultar** no vazamento de dados sensíveis._"

Reply:
fix
* where it's written "_Atacantes **ter** acesso a dados sensíveis (...)_"
* it should be "_Atacantes **têm** acesso a dados sensíveis (...)_